ts/utils/brightness.ts

```diff
@@ -26,48 +27,67 @@ export const setBrightness = (brightness: number): TaskEither<Error, number> =>
     reason => new Error(String(reason))
   );
 
-const HIGH_BRIGHTNESS = 1.0; // Target screen brightness for a very bright screen
+const HIGH_BRIGHTNESS = 1.0; // Max brightness value
 
 /**
- * Hook that sets the brightness to maximum, until the screen is unmounted
+ * Set the device brightness to the max value and restore the original brightness when
+ * the component is unmount or the app changes state (!== active)
  */
 export const useMaxBrightness = () => {
-  // Brightness effect manager
-  React.useEffect(() => {
-    // eslint-disable-next-line functional/no-let
-    let myBrightness: number | undefined;
+  // The initial device brightness
+  const [startBrightness, setStartBrightness] = useState<number | undefined>(
```

Comment:
If we rename this to `initialBrightness` it's easier to read down the line and you can get rid of the comment above :-)

Reply:
You're right! Changed in ffd18ec5bfa3d0212a9a5ef8528ed355a40f8caf